run_dir/static/js/worksets.js

```diff
@@ -118,10 +118,16 @@ $(".tabbable").on("click", '[role="tab"]', function() {
                 to_return = to_return +'</table></span>';
                 return to_return;
               }));
-            tbl_row.append($('<td>').html(projval['samples'].length +' ('+ projval['total_num_samples']+')'));
+            tbl_row.append($('<td>').html(projval['samples'].length +' <span class="badge">'+ projval['total_num_samples']+'</span>'));
             sumGroups[key] = sumGroups[key] + projval['samples'].length;
             var number_of_days = Math.floor(Math.abs(new Date() - new Date(projval['queued_date']))/(1000*86400));
-            tbl_row.append($('<td>').html(number_of_days));
+            var lblinf='';
+            switch(Math.floor(number_of_days/7)){
```

Comment:
Maybe we can put this in to a function - at the moment there are three slightly different implementations to do the same thing, I could imagine it being easy to miss something if we want to change it in the future.

Reply:
I put it in a function in all 3 files.